run-tests.sh

```diff
@@ -13,17 +14,6 @@ set -o errexit
 # Quit on unbound symbols
 set -o nounset
 
-# Always bring down docker services
```

Comment:
were the docker services unnecessary?

Reply:
As you see the tests are running through. It would be really weird if the tests were green but the services were necessary.

Reply:
good point! :smile:

Reply:
this will have an impact locally though, it will require manual run of docker container before starting the tests, which is not consistent with other modules

Reply:
If that would have an impact locally the tests on github would also not work, but they are passing. For me that shows that the running docker containers are not necessary for the tests of this package. Further i tried another time locally to be sure that no containers are up and running. It worked locally without running containers.

Reply:
readded